hieradata/nodes/cyclone.yaml

```diff
@@ -0,0 +1 @@
+pulse_sink: alsa_output.usb-C-Media_Electronics_Inc._USB_Audio_Device-00.analog-stereo
```

Comment:
I'd document where you got these from, if they ever need to be updated or changed in any way.